docs/blog/12-production-deployment.md

```
@@ -748,6 +809,10 @@ spec:
           periodSeconds: 30
       selectPolicy: Max
 ```
+Expected output:
+```text
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)